src/components/WalletDropdown/IconButton.tsx

```diff
@@ -0,0 +1,67 @@
+import { PropsWithChildren } from 'react'
+import { Icon } from 'react-feather'
+import styled from 'styled-components/macro'
+
+export const IconHoverText = styled.span`
+  color: ${({ theme }) => theme.textPrimary};
+  position: absolute;
+  top: 28px;
+  border-radius: 8px;
+  transform: translateX(-50%);
+  opacity: 0;
+  font-size: 12px;
+  padding: 5px;
+  left: 10px;
+`
+
+interface BadgeProps {
+  onClick: () => void
+}
+
+const IconBlock = styled.div<PropsWithChildren<BadgeProps>>`
+  background-color: ${({ theme }) => theme.backgroundInteractive};
+  border-radius: 12px;
+  display: inline-block;
+  position: relative;
+  height: 32px;
+  width: 32px;
+  color: ${({ theme }) => theme.textPrimary};
+  :hover {
+    background-color: rgba(173, 188, 255, 0.48);
+    -webkit-transition: background-color 200ms linear;
+    cursor: pointer;
+    ${IconHoverText} {
+      opacity: 1;
+    }
+  }
+  :active {
+    background-color: ${({ theme }) => theme.backgroundSurface};
+    -webkit-transition: background-color 50ms linear;
```

Comment:
we have `autoprefixer` in our build process, so maybe don't need webkit specific overrides?